expression/integration_test.go

```diff
@@ -7298,6 +7295,21 @@ func (s *testIntegrationSuite) TestIssue19596(c *C) {
 	tk.MustGetErrMsg("create table t (a int) partition by range(a) (PARTITION p0 VALUES LESS THAN (a));", "[expression:1054]Unknown column 'a' in 'expression'")
 }
 
+func (s *testIntegrationSerialSuite) TestIssue11645(c *C) {
```

Comment:
I think using parallel test is enough....

Reply:
Thanks for the advise.